go.mod

```
@@ -10,7 +10,7 @@ require (
 	github.com/go-logr/logr v1.4.1
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/go-cmp v0.6.0
-	github.com/humio/cli v0.35.1
+    github.com/humio/cli v0.36.0
```

```suggestion
	github.com/humio/cli v0.36.0
```

I fixed this with the github conflict resolution UI but it messed up the formatting.